docs/installation/installation.md

```diff
@@ -185,9 +185,15 @@ kubectl create secret -n tyk-operator-system generic tyk-operator-conf \
 If you don't have cert-manager installed, here is a quick install:
 
 ```bash
-kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.0.3/cert-manager.yaml
+kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.3.0/cert-manager.yaml
 ```
 
+This cert-manager version is the minimum version that works with kubernetes version 1.22 . 
```

Comment:
Is cert-manager `v1.3.0` compatible with Kubernetes 1.22? In the cert-manager doc, `v1.3.0` is compatible with Kubernetes v1.16 and 1.21 https://cert-manager.io/docs/installation/supported-releases/#old-releases

Reply:
I have tested this manually and cert-manager v1.3.0 works with our operator running on a kubernetes 1.22.5 cluster.